src/forms/components/encounter/ohri-encounter-form.tsx

```diff
@@ -235,16 +259,33 @@ export const OHRIEncounterForm: React.FC<OHRIEncounterFormProps> = ({
       const isHidden = eval(hideExpression);
       if (field) {
         field.isHidden = isHidden;
+
+        //evaluate markdown visibility
+        if (field.markdown?.hide) {
+          field.markdown.isHidden = eval(field.markdown.hide);
+        }
       }
       if (page) {
         page.isHidden = isHidden;
+
+        // evaluate markdown visibility
+        if (page.markdown?.hide) {
+          page.markdown.isHidden = eval(page.markdown.hide);
+        }
+
         page.sections.forEach(section => {
           section.isParentHidden = isHidden;
           cascadeVisibityToChildFields(isHidden, section, allFields);
         });
       }
       if (section) {
         section.isHidden = isHidden;
+
+        //evaluate markdown visibility
+        if (section.markdown?.hide) {
+          section.markdown.isHidden = eval(section.markdown.hide);
```

Comment:
@larslemos I thought markdowns are just fields or questions that don post any obs?